lib/analytical/modules/segment.rb

```diff
@@ -0,0 +1,130 @@
+module Analytical
+	module Modules
+		class Segment
+			include Analytical::Modules::Base
+			
+			def initialize(options={})
+				super
+				@tracking_command_location = :body_append
+			end
+
+			def init_javascript(location)
+			init_location(location) do
```

Comment:
The appropriate indentation for this method would be something along the lines of:
```ruby
def init_javascript(location)
  init_location(location) do
```